hail/python/hail/utils/struct.py

```diff
@@ -143,6 +143,14 @@ def select(self, *fields, **kwargs):
         -------
         :class:`.Struct`
             Struct containing specified existing fields and computed fields.
+
+        Examples
+        --------
+
+        Define s
+        >>> s = hl.Struct(**{'foo': 5, '1kg': 10})
```

Comment:
On your previous PR I said we should avoid `**` syntax. I still think we should avoid it but, you're using an example of the kind of Struct field name that *must* use the `**` syntax:
```
hl.Struct(**{'foo': 5, '1kg': 10})
```
Python rejects this expression:
```
hl.Struct(foo=5, 1kg=10)
```
```
In [1]: hl.Struct(foo=5, 1kg=10) 
   ...:                                                                                                                                                       
  File "<ipython-input-1-611e92d5a31f>", line 1
    hl.Struct(foo=5, 1kg=10)
                       ^
SyntaxError: invalid syntax
```

This is because python does not allow us to user variable names which start with a digit. `1kg` starts with a `1` so its invalid.

---

Let's change the field name from `1kg` to `bar` or `apple` or w/e. Anything without a leading digit. If you want to have two examples one without `**` and one *with* `**`, then I'm OK with that! I just want the *first* example to be an easy one.

Reply:
Addressed changes